core/bin/external_node/src/main.rs

```diff
@@ -348,10 +349,13 @@ async fn main() -> anyhow::Result<()> {
         .main_node_url()
         .context("Main node URL is incorrect")?;
 
-    let connection_pool = ConnectionPool::builder(DbVariant::Master)
-        .build()
-        .await
-        .context("failed to build a connection_pool")?;
+    let connection_pool = ConnectionPool::builder(
+        config.postgres.database_url.clone(),
+        config.postgres.max_connections,
```

Comment:
how about taking the whole config as an argument?

Reply:
This function is invoked using different configs for server/EN, and for the server, we have multiple different URLs.
I didn't find a way to introduce a reusable struct here that would be better than just passing two variables.

Reply:
nvm, there are different PostgresConfig types